modules/configuring-quota-management-for-distributed-workloads.adoc

```diff
@@ -0,0 +1,133 @@
+endif::[]
+ifdef::self-managed[]
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+endif::[]
+ifdef::self-managed[]
+* You have downloaded and installed the OpenShift command-line interface (CLI). See link:https://docs.openshift.com/container-platform/{ocp-latest-version}/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI].
+endif::[]
+ifdef::cloud-service[]
+* You have downloaded and installed the OpenShift command-line interface (CLI). See link:https://docs.openshift.com/dedicated/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI] (Red Hat OpenShift Service on AWS).
+endif::[]
+* You have sufficient resources. In addition to the base {productname-short} resources, you need 1.1 vCPU and 1.6 GB memory to deploy the distributed workloads infrastructure.
+* The resources are physically available in the cluster.
++
+[NOTE]
+====
+{productname-short} currently supports only a single cluster queue per cluster (that is, homogenous clusters), and only empty resource flavors.
+For more information about Kueue resources, see the link:https://kueue.sigs.k8s.io/docs/concepts/[Kueue documentation].
+====
+
+
+.Procedure
+
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<admin_username>__ -p __<password>__
+----
+
+. Create an empty Kueue resource flavor, as follows:
+.. Create a file called `default_flavor.yaml` and populate it with the following content:
++
+.Empty Kueue resource flavor
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: default-flavor
+----
+.. Apply the configuration to create the `default-flavor` object:
++
+[source,bash]
+----
+$ oc apply -f default_flavor.yaml
+----
+
+. Create a cluster queue to manage the empty Kueue resource flavor, as follows:
+.. Create a file called `cluster_queue.yaml` and populate it with the following content:
++
+.Example cluster queue
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "cluster-queue"
+spec:
+  namespaceSelector: {}  # match all.
+  resourceGroups:
+  - coveredResources: ["cpu", "memory", "nvidia.com/gpu"]
+	flavors:
+	- name: "default-flavor"
+  	  resources:
+   	  - name: "cpu"
+    	nominalQuota: 9
+  	  - name: "memory"
+    	nominalQuota: 36Gi
+  	  - name: "nvidia.com/gpu"
+    	nominalQuota: 5
+----
++
+.. Replace the example quota values (9 CPUs, 36 GiB memory, and 5 NVIDIA GPUs) with the appropriate values for your cluster queue.
```

Comment:
Is this only supported with NVIDIA GPUs? What about Habana Gaudi devices?

Reply:
Not sure, will check with QE.

Reply:
Added a note to say that only NVIDIA GPUs are supported in this release.